modules/Kconfig.cmsis

```diff
@@ -28,3 +28,13 @@ menuconfig CMSIS_DSP
 if CMSIS_DSP
 source "modules/Kconfig.cmsis_dsp"
 endif
+
+menuconfig CMSIS_NN
+	bool "CMSIS-NN Library Support"
+	depends on CPU_CORTEX
+	depends on NEWLIB_LIBC
```

Comment:
Does CMSIS-NN directly depend on the functions that are not available in the Zephyr minimal libc?

If not, `depends on NEWLIB_LIBC` should be removed (note that `select CMSIS_DSP`, which does depend on the newlib, will resolve the dependency anyway).